x-pack/plugins/fleet/server/services/package_policies_to_agent_permissions.ts

```diff
@@ -11,21 +11,7 @@ import { PACKAGE_POLICY_DEFAULT_INDEX_PRIVILEGES } from '../constants';
 import { getPackageInfo } from '../../server/services/epm/packages';
 import type { PackagePolicy } from '../types';
 
-export const DEFAULT_PERMISSIONS = {
-  cluster: ['monitor'],
-  indices: [
-    {
-      names: [
-        'logs-*',
-        'metrics-*',
-        'traces-*',
-        'synthetics-*',
-        '.logs-endpoint.diagnostic.collection-*',
-      ],
-      privileges: PACKAGE_POLICY_DEFAULT_INDEX_PRIVILEGES,
-    },
-  ],
-};
+export const DEFAULT_CLUSTER_PERMISSIONS = ['monitor'];
```

Comment:
Just for my info, why do agents need the monitor cluster privilege? Cluster health?

Reply:
Yes beats is checking the cluster health

Reply:
If I remember correctly, this is because Beats is checking the version of ES but something that we should do in a different way. I tried to find the beats issue but couldn't quickly. I remember pining @jlind23 on this, maybe he has it handy. As soon as this is fixed in beats, we should remove these permissions here too.